src/bank.rs

```diff
@@ -212,6 +214,15 @@ impl Bank {
         }
     }
 
+    fn check_last_id_age(last_id_queue: &VecDeque<Hash>, last_id: Hash) -> usize {
```

Comment:
The codebase uses `check` as a synonym to `verify`, not as a shorthand for `go_check_on`. Can you change it to `get`? Also, consider changing `last_id` to `entry_id` in both the function name and parameter name.